addons/l10n_mx/models/chart_template.py

```diff
@@ -17,6 +17,8 @@ def _load_template(
             company, code_digits=code_digits,
             transfer_account_id=transfer_account_id, account_ref=account_ref,
             taxes_ref=taxes_ref)
+        if not self.env.user.company_id.country_id == self.env.ref('base.mx'):
```

Comment:
No, that's not true for the chart template installation, that can be done from Settings menu under Accounting. Anyway, I have already told you how to check this properly in other comment.